guess_a_no/README.md

```diff
@@ -0,0 +1,37 @@
+Game: Guess A Number
+=======
+
+### Objective
+
+Objective is to build a fun game, which will help a User to get familar with random no and basic logic. It's a small script and any language can be used for coding.
+

```

Comment:
Graphics is just a suggestion. I am more in favour of adding some kind of theme. I thought our main interest is to spike up the interest of user.